infoschema/tables.go

```diff
@@ -797,6 +797,31 @@ func dataForColumnsInTable(schema *model.DBInfo, tbl *model.TableInfo) [][]types
 		if colLen == types.UnspecifiedLength {
 			colLen = defaultFlen
 		}
+		if col.Tp == mysql.TypeSet {
+			// Example: In MySQL set('a','bc','def','ghij') has length 13, because
+			// len('a')+len('bc')+len('def')+len('ghij')+len(ThreeComma)=13
+			// Reference link: https://bugs.mysql.com/bug.php?id=22613
+			sumLen, cnt := 0, 0
```

Comment:
how about this:
```go
colLen = 0
for _, ele := range col.Elems {
	colLen += len(ele)
}
if len(col.Elems) != 0 {
	colLen = len(col.Elems) - 1
}
```

Reply:
@crazycs520 
it looks grate. But it should be:  

    if len(col.Elems) != 0 {
	    colLen += (len(col.Elems) - 1)
    }

Reply:
ye~😢